### ISSUE TITLE
"Visible Section Not Found" message appears if Magento admin session times out

### ISSUE THREAD

**Issue by hellotaran** [magento/magento2#18346]
### Preconditions

1. Magento 2.3-develop branch
2. PHP 7.1.22
3. Ubuntu 14.04.1
4. Installed on Vagrant Machine
5. Default "Admin Session Lifetime (seconds)" = 900
6. Google Chrome Version 69.0.3497.100 (Official Build) (64-bit)
7. Base Machine Windows 10 Pro

### Steps to reproduce

1. Login to Magento admin and wait for 900 seconds or 15 minutes.
2. Post 15 minutes click on any menu link in admin area.
3. Magento admin user get redirected to login screen with a message "Visible Section Not Found". 

### Expected result

1. Message must be "You have logged out".

![image](https://user-images.githubusercontent.com/7958215/46370517-cbe27000-c653-11e8-8256-2a583a56acc5.png)

### Actual result

1. Currently it appears like this.

![image](https://user-images.githubusercontent.com/7958215/46370553-de5ca980-c653-11e8-8c26-40d70e45b57d.png)

**Comment by magento-engcom-team** [comment]
Hi @hellotaran. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@hellotaran do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by fooman** [comment]
Maybe while this is being looked at the message can be improved to:
"You were logged out due to inactivity."

"You have logged out" implies an active action by the user.

**Comment by magento-engcom-team** [comment]
Hi @dthampy. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@dthampy Thank you for verifying the issue.

Unfortunately, not enough information was provided to created internal ticket. Please consider adding the following:
- [ ] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess
- [ ] Add `"Reproduced on "` label(s) to this ticket based on verification result

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by dthampy** [comment]
Internal ticket MAGETWO-95482 has been created. Thank you for your submission.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by dmytro-ch** [comment]
The error message should be generated by the `\Magento\Security\Model\AdminSessionsManager::getLogoutReasonMessageByStatus` method.
However, it seems not to be called in an expected way.

**Comment by CuongPhi** [comment]
I created custom module with incorrect name and got this issue.
Sr, bad english

**Comment by ghost** [comment]
@magento i am working on it

**Comment by ghost** [comment]
Fixed!
![image](https://user-images.githubusercontent.com/56463897/66700013-c3625200-ed09-11e9-98f2-e0c1ef2af8e7.png)

**Comment by engcom-Charlie** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @engcom-Charlie. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @engcom-Charlie, here is your [Magento instance](https://i-18346-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-18346-2-4-develop.instances.magento-community.engineering/admin](https://i-18346-2-4-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by engcom-Charlie** [comment]
Hello @hellotaran 

I am not able to reproduce this issue on a fresh Magento 2.4-develop.

**Testing scenario:**

1. Login to Magento admin and wait for 900 seconds or 15 minutes.
2. Post 15 minutes click on any menu link in admin area.

**Result:**
No any message. 
![image](https://user-images.githubusercontent.com/51680850/71514082-c0591580-28a5-11ea-86e8-79fc6e8cc5e4.png)

So i have to close this issue as not relevant.

Please feel free to comment, reopen or create new ticket according to the Issue reporting guidelines
if you are still facing this issue on the latest Magento 2.4-develop branch.

Thanks for your report!
